### DISCUSSION TITLE
How to disable one block claim

### DISCUSSION THREAD

**Discussion by Raidware** [GriefPrevention/GriefPrevention#2313] [Q&A]
i saw my player claimed one block. How to disable?

**Comment by Jikoo** [comment]
`GriefPrevention.Claims.MinimumWidth` and `GriefPrevention.Claims.MinimumArea` control minimum claim sizes.
It also may be possible that they placed a chest next to another claim for their starter claim, which I believe caused very small claims on some versions. Unsure if that behavior still exists.

**Reply by Raidware** [reply]
> `GriefPrevention.Claims.MinimumWidth` and `GriefPrevention.Claims.MinimumArea` control minimum claim sizes. It also may be possible that they placed a chest next to another claim for their starter claim, which I believe caused very small claims on some versions. Unsure if that behavior still exists.

i changedt them both to 100

**Reply by Jikoo** [reply]
As long as that works for you, excellent, but having a minimum width of 100 means a minimum area of 10,000 because the claim must be 100x100. Defaults are 5 for width to allow road-style claims, 100 for area.

**Reply by Raidware** [reply]
> As long as that works for you, excellent, but having a minimum width of 100 means a minimum area of 10,000 because the claim must be 100x100. Defaults are 5 for width to allow road-style claims, 100 for area.

hi how can i fix this? my players cannot claim

**Reply by Jikoo** [reply]
Like I said, because you set the minimum width to 100, you require claims to be 100x100 minimum, a total area of at least 10,000. The default minimum width is 5. Combined with the default minimum area of 100, that would mean that claims would have to be at least 5x20, 6x17, and so on up to 10x10.

**Reply by Jikoo** [reply]
For clarity, a better description of MinimumWidth might be minimum side length of both X and Z dimensions.

**Reply by Raidware** [reply]
> For clarity, a better description of MinimumWidth might be minimum side length of both X and Z dimensions.

but how can i disable chest claiming  Because players can claim only 1 block.

**Reply by bigpresh** [reply]
> but how can i disable chest claiming

See `Claims.AutomaticNewPlayerClaimsRadius` in the docs: https://docs.griefprevention.com/configuration/#land-claim-settings
